style/global/_typography.scss

```diff
@@ -0,0 +1,79 @@
+$base-font: 'Inconsolata', 'Courier New', Courier, monospace;
+$secondary-font: 'Prata', sans;
```

Comment:
You could use some other system font as a fallback here. Btw it should be `sans-serif`, not `sans`. 😄 Take a look which fonts are safe to declare: https://www.cssfontstack.com/